packages/block-renderer/src/components/pattern-renderer.tsx

```diff
@@ -20,19 +20,32 @@ const PatternRenderer = ( {
 	viewportHeight,
 	minHeight,
 	maxHeight,
-	prependHtml = '',
+	transformHtml,
 	shouldShufflePosts,
 }: Props ) => {
 	const renderedPatterns = usePatternsRendererContext();
 	const pattern = renderedPatterns[ patternId ];
-	const patternHtml = usePatternMinHeightVh( prependHtml + pattern?.html, viewportHeight );
-	const inlineCss = usePatternInlineCss( patternId, patternHtml, shouldShufflePosts );
+
+	if ( ! pattern ) {
+		return null;
```

Comment:
this will make the placeholder height is not able to show when loading the pattern like below

![image](https://github.com/Automattic/wp-calypso/assets/13596067/9e6978af-b484-4926-ab5b-53e03076d133)

Reply:
Ah, interesting... I did this because the type definition of `RenderedPattern` has non-optional fields... got tripped by TypeScript 😂 

https://github.com/Automattic/wp-calypso/blob/dc04b76f80b8dcfa884127a8568d3243b2e330cb/packages/block-renderer/src/types.ts#L11-L17

Will fix this, thanks!

Reply:
Reverted for now, but I think we should rethink the type definition later 😄